database/kwdb.sql

```diff
@@ -20,6 +20,30 @@ CREATE TABLE IF NOT EXISTS "command_label" (
   PRIMARY KEY("id")
 );
 
+-- Table containing the kw_mail groups 
+CREATE TABLE IF NOT EXISTS "groups" (
+    "id" INTEGER NOT NULL UNIQUE,
+    "name" VARCHAR(50) NOT NULL UNIQUE,
```

Comment:
If we limit this, ensure that our code that interacts with this table validates the maximum size.